eth/stagedsync/stage_execute.go

```diff
@@ -768,10 +768,14 @@ func PruneExecutionStage(s *PruneState, tx kv.RwTx, cfg ExecuteBlockCfg, ctx con
 			if err = rawdb.PruneTable(tx, kv.BorReceipts, cfg.prune.Receipts.PruneTo(s.ForwardProgress), ctx, math.MaxUint32); err != nil {
 				return err
 			}
-			// LogIndex.Prune will read everything what not pruned here
-			if err = rawdb.PruneTable(tx, kv.Log, cfg.prune.Receipts.PruneTo(s.ForwardProgress), ctx, math.MaxInt32); err != nil {
+
+			const maxPruneDelta = 100 // at most 100 blocks will be pruned at once
+			// LogIndex.Prune will read everything what not pruned here, limited
```

Comment:
problem is not in limiting. problem is in this comment line.
see: eth/stagedsync/stage_log_index.go `pruneLogIndex`
It does iterate over all data: 
`for k, v, err := c.First(); k != nil; k, v, err = c.Next() {`

I don't really remember:
- or `pruneLogIndex` become very slow because not all data pruned here
- or `pruneLogIndex` do not prone enough

But I remember that when I put limit here `pruneLogIndex` caused some problems.